easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonBundle'
+
+name = 'Keras'
+version = '2.2.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://keras.io/'
+description = """Keras is a minimalist, highly modular neural networks library, written in Python and
+capable of running on top of either TensorFlow or Theano."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('TensorFlow', '1.8.0', versionsuffix),
+    ('Theano', '1.0.2', versionsuffix),
+    ('h5py', '2.7.1', versionsuffix),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+
+use_pip = True
+
+exts_list = [
+    ('Keras-Applications', '1.0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Applications'],
+        'source_tmpl': 'Keras_Applications-%(version)s.tar.gz',
+        'checksums': ['a03af60ddc9c5afdae4d5c9a8dd4ca857550e0b793733a5072e0725829b87017'],
+    }),
+    ('Keras-Preprocessing', '1.0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Preprocessing'],
+        'source_tmpl': 'Keras_Preprocessing-%(version)s.tar.gz',
+        'checksums': ['ef2e482c4336fcf7180244d06f4374939099daa3183816e82aee7755af35b754'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras'],
+        'checksums': ['90b610a3dbbf6d257b20a079eba3fdf2eed2158f64066a7c6f7227023fd60bc9'],
+    }),
+]
+
+sanity_check_paths = {
```

Comment:
Isn't this particular sanity check also moot with the new `PythonBundle`?

Reply:
Ah, right, yes it is, this is exactly the default used by `PythonPackage`... Cleaning up...